codec-http/src/test/java/io/netty/handler/codec/http/HttpContentCompressorTest.java

```diff
@@ -217,6 +219,65 @@ public void testChunkedContentWithAssembledResponse() throws Exception {
         assertThat(ch.readOutbound(), is(nullValue()));
     }
 
+    @org.junit.Test
```

Comment:
Wrong annotation type
```suggestion
    @Test
```